src/makeRobot.js

```diff
@@ -38,7 +38,74 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return 'name: ' + `${this.name}, `
+      + 'chip version: ' + `${this.version}, `
+      + 'wheels: ' + `${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(point = 1) {
+      if (point < 0) {
+        return robot;
+      }
+
+      this.coords.y += point;
```

Comment:
```suggestion
     if(point > 0) {
         this.coords.y += point;
     }
```

better add if statement block to this and fix it for each function